test/cypress/support/output-logs.js

```diff
@@ -3,18 +3,23 @@
 // Copied directly from: https://github.com/cypress-io/cypress/issues/3199#issuecomment-466593084
 // ***********
 
+const APPLICATION_NAME = 'group-income'
```

Comment:
Will merge but in the future we should get this using `JSON.parse(fs.readFileSync(pathToPkgJson)).name`

Reply:
That's a better way indeed. Or `require(pathToPkgJson).main` if we can

Reply:
Oh yeah, I totally forgot that `require` might be able to do that